src/menu_parser.py

```diff
@@ -217,6 +217,21 @@ def get_menus(self, text, year, week_number):
             lines_weekdays["thu"] += " " + line[pos_thu:pos_fri].replace("\n", " ").replace("Donnerstag", "")
             lines_weekdays["fri"] += " " + line[pos_fri:].replace("\n", " ").replace("Freitag", "")
 
+        # Add the aktionsgericht
+        line_aktion = [s for s in lines if "Aktion" in s]
+        num_dishes = 3
+        if len(line_aktion) == 1:
+            num_dishes = 4
+            line_aktion_pos = lines.index(line_aktion[0]) - 1
+            aktionsgericht = ' '.join(lines[line_aktion_pos:line_aktion_pos + 3])
+            aktionsgericht = aktionsgericht \
+                .replace("Montag – Freitag", "") \
+                .replace("Tagessuppe täglich wechselndes Angebot", "") \
+                .replace("ab € 1,00", "")
+            for key in lines_weekdays:
```

Comment:
You could probably use map here which is a bit more beautiful.

Reply:
```
lines_weekdays = dict(map(lambda x: aktionsgericht + ", " + x, lines_weekdays))
```
but this will get rid of the keys. I tried several variations...